roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -153,6 +153,9 @@ kube_apiserver_port: 6443
 kube_apiserver_insecure_bind_address: 127.0.0.1
 kube_apiserver_insecure_port: 0
 
+kube_controller_manager_bind_address: 127.0.0.1
```

Comment:
Since it looks like these vars are only used in the `kubernetes/master` role I think you should move these defaults to `roles/kubernetes/master/defaults/main.yml` 

If you're feeling more ambitious you could alternatively refactor all of the vars in the spec into `roles/kubernetes/master/defaults/kube_controller_manager.yml` and `roles/kubernetes/master/defaults/kube_scheduler.yml` like we did in #3958 for kube-proxy.

Reply:
Moved the variables to `roles/kubernetes/master/defaults/main/main.yml` for now. A refactor of all the vars in the spec should, in my opinion, be a separate PR to bring complexity down and better readability.